Analysis/src/EventInfo.cpp

```diff
@@ -347,13 +366,62 @@ const MET& EventInfoBase::GetMET()
     return *met;
 }
 
-const size_t EventInfoBase::GetLegIndex(const size_t leg_id)
+size_t EventInfoBase::GetLegIndex(const size_t leg_id)
 {
     if(leg_id == 1) return event->first_daughter_indexes.at(selected_htt_index);
     if(leg_id == 2) return event->second_daughter_indexes.at(selected_htt_index);
     throw exception("Invalid leg id = %1%.") % leg_id;
 }
 
+bool PassedLegSelection(const ntuple::TupleLepton& lepton, const Channel& channel)
```

Comment:
- `PassedLegSelection` -> `PassDefaultLegSelection`
- `const Channel&` -> `Channel`
- should be declared in .h